src/routes/controllers/project.controller.js

```diff
@@ -58,6 +58,33 @@ const createProject = async (req, res, next) => {
   }
 };
 
+const getProject = async (req, res, next) => {
+  try {
+    const { projectId } = req.params;
+    const { _id: userId } = req.user;
+
+    if (!mongoose.isValidObjectId(projectId)) {
+      const error = createError(400, BAD_REQUEST);
+      logger.error(error.toString());
+
+      return next(error);
+    }
+
+    const project = await Project.findOne({ _id: projectId, owner: userId })
```

Comment:
```suggestion
    const project = await Project
      .findOne({ _id: projectId, owner: userId })
```
줄을 바꿔서 체이닝을 이어서 해나가는건 어떨까요

Reply:
완전 좋죠!! 저도 체이닝시 내려쓰는걸 선호하는 편이라, prettier 앞으로 적용하지 않기로 하였으니 기준만 정하면 될 것 같아요 ㅎㅎ 첫 메소드부터 바로 체이닝 하는 것으로 다들 괜찮으실까요??

Reply:
네넵! 첫 메서드부터 체이닝 좋습니다!

Reply:
노션 최하단에 추가해뒀습니다~!